contracts/foundry.toml

```diff
@@ -92,11 +92,23 @@ src = 'src/v0.8/operatorforwarder'
 test = 'src/v0.8/operatorforwarder/test'
 
 [profile.workflow]
```

Comment:
You could consider a separate compile profile that is used for wrapper generation. That one then uses via-ir while the main one devs use during development uses fast compilation. It can also have low optimizer runs to speed it up further. This is what we do with CCIP as compiling with via-ir takes ages. 

What you currently have is totally fine as well!

Reply:
:D valid suggestions, though, this contract is much smaller. Recent ran got `Solc 0.8.24 finished in 1.86s` so I guess that might shave off a second

Reply:
You will develop more, including more tests which will add a lot of time in the future, right? Your choice either way

Reply:
Is this still applicable though if we need the tag due to too many local vars in one of the function